src/Analysis/Ast/Impl/Analyzer/Handlers/FromImportHandler.cs

```diff
@@ -82,7 +82,7 @@ internal sealed partial class ImportHandler {
             }
 
             // If __all__ is present, take it, otherwise declare all members from the module that do not begin with an underscore.
-            var memberNames = variableModule.Analysis.StarImportMemberNames ?? variableModule.GetMemberNames().Where(s => !s.StartsWithOrdinal("_"));
+            var memberNames = variableModule.Analysis?.StarImportMemberNames ?? variableModule.GetMemberNames().Where(s => !s.StartsWithOrdinal("_"));
```

Comment:
Should module have at least empty analysis always? I think there is a lot of code that assumes that.